aten/src/ATen/native/quantized/cpu/qlinear.cpp

```diff
@@ -153,10 +159,10 @@ class QLinearInt8 final : public c10::OperatorKernel {
 
 static auto registry =
     c10::RegisterOperators()
-        .op("quantized::fbgemm_linear(Tensor X, Tensor W_prepack, Tensor b, float Y_scale_i, int Y_zero_point_i) -> Tensor Y",
+        .op("quantized::fbgemm_linear(Tensor X, Tensor W_prepack, Tensor? b=None, float Y_scale_i, int Y_zero_point_i) -> Tensor Y",
```

Comment:
@smessmer : For C10 Op registration, it doesn't support None for the arguments, is that right? ATen has None Tensor support, e.g., https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml#L1078

Reply:
C10 does support none, just make your argument c10::optional.

Reply:
Fixed. Thanks @smessmer !